chore(web-vitals): organizing sections, updating thresholds and adding browser support (#3814)
getsentry/sentry-docs: src/docs/product/performance/web-vitals.mdx

| --------------------------------------------------------------- | ------------ | ------------ | ------------ |
| [Largest Contentful Paint](#largest-contentful-paint-lcp) (LCP) | <= 2.5s      | <= 4s        | > 4s         |
| [First Input Delay](#first-input-delay-fid) (FID)               | <= 100ms     | <= 300ms     | > 300ms      |
| [Cumulative Layout Shift](#cumulative-layout-shift-cls) (CLS)   | <= 0.1       | <= 0.25      | > 0.25       |
| [First Paint](#first-paint-fp) (FP)                             | <= 1s        | <= 3s        | > 3s         |
| [First Contentful Paint](#first-contentful-paint-fcp) (FCP)     | <= 1s        | <= 3s        | > 3s         |
| [Time To First Byte](#time-to-first-byte-ttfb) (TTFB)           | <= 100ms     | <= 200ms     | > 600ms      |

<Note>

Some Web Vitals such as FP, FCP, LCP, and TTFB are measured relative to the start of the transaction. Values may differ when compared to values generated with other tools such as [Lighthouse](https://github.com/GoogleChrome/lighthouse).

</Note>

## Distribution Histogram

![Example of Web Vitals](./perf-web-vitals.png)

> A histogram visualization showing the data distribution can help you identify and diagnose frontend performance problems by revealing anomalies.

Reviewer: ```suggestion
The Web Vitals histogram displays data distribution, and it can help you identify and diagnose frontend performance problems by revealing anomalies.
```
